### ISSUE TITLE
missing models

### ISSUE THREAD

**Issue by haimmm** [OlafenwaMoses/ImageAI#475]
I'm trainint atm my own data with batch size of 16 and 150 experiments using google colab connected to drive.

however, its already at 74/150 but in the models i directory i see like:

"1, 2, 3,...,20, 24, 35, 37, 43" and thats it.

where's all the others? shouldn't i have like 74 files already?

**Comment by Gytis123** [comment]
Yes it saves only those which are improved. if model accuracy reaches 1.00.. then it should stop training
